projects/JuMP.md

```diff
@@ -24,13 +24,19 @@ If you find JuMP useful in your work, we kindly request that you cite the follow
 ## Governance
 
 ### Leadership Team
-?
+
+- Miles Lubin (Google)
+- Juan Pablo Vielma (MIT)
+- Joey Huchette (Google)
+- Oscar Dowson (Northwestern)
+- Changhyun Kwon (U. of South Florida)
 
 ### Code of Conduct
 https://github.com/JuliaOpt/JuMP.jl/blob/master/CODE_OF_CONDUCT.md
 
 ### Governance Documentation
-?
+
+N.A.
```

Comment:
N.A. might be interpreted as implying that governance documentation doesn't apply to us :). I'd say "None (yet)".